Makefile

```diff
@@ -1,9 +1,9 @@
 DESTDIR =
 PREFIX  = /usr/local
 
-BASHDIR = /etc/bash_completion.d
+BASHDIR = $(PREFIX)/etc/bash_completion.d
 ZSHDIR  = /usr/share/zsh/vendor-completions
```

Comment:
By "these directories" do you mean all 3 of them?

Regarding bash, on macOS, `/etc/bash_completion.d` doesn't even exist, and `/usr/local/etc/bash_completion.d` is where various third-party utilities have installed their completions. That said, I don't use bash these days, and poking around in its init files, it actually looks to me as though Bash 3.2.57 on macOS doesn't actually have a default completions directory and users are expected to handle sourcing the files themselves. Either way, third-party software is definitely not supposed to be installing anything into `/etc`, especially when the software itself lives in `/usr/local`.

As for Fish, make sure you're using a recent version. `/usr/local/share/fish/vendor_completions.d` was added in Fish 2.2. Also FWIW, it looks as though Fish's pkg-config actually vends a path for this using `pkg-config --variable completionsdir fish`, so we should consider using that if pkg-config exists, but maybe do that as a followup PR?